internal/pkg/cli/task_run.go

```diff
@@ -1193,6 +1199,7 @@ func BuildTaskRunCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&vars.groupName, taskGroupNameFlag, nameFlagShort, "", taskGroupFlagDescription)
 
 	cmd.Flags().StringVar(&vars.dockerfilePath, dockerFileFlag, defaultDockerfilePath, dockerFileFlagDescription)
+	cmd.Flags().StringToStringVar(&vars.dockerfileBuildArgs, dockerFileBuildArgsFlag, nil, dockerFileBuildArgsFlagDescription)
```

Comment:
Can you add an example to the [task run help menu] about how we can run task from dockerfile 🙏

Reply:
I added an example to the help menu showing how to use the --build-args flag. Thanks for the suggestion!